CommandsFunctions/Create.py

```diff
@@ -0,0 +1,36 @@
+import os
+import json
+from SchemaKeys import Keys
+from Icommand import Icommand
+
+class Create(Icommand):
+    def __init__(self, schema_file):
```

Comment:
too much try and catch isn't good. 
it is better to have validate function. It throws exception in case anything error, then execute your code

Reply:
look at @bangkook code as an example. I did a comment in this part there